pkgs/applications/blockchains/sparrow/default.nix

```diff
@@ -5,7 +5,7 @@
 , makeDesktopItem
 , copyDesktopItems
 , autoPatchelfHook
-, openjdk18
+, openjdk19
```

Comment:
In #199802, I'm making 19 the default, so you could just switch it to use `openjdk` (unversioned).  That way it'll get future security updates automatically.